package.json

```diff
@@ -30,6 +30,7 @@
   "dependencies": {
     "@govuk-one-login/di-ipv-cri-common-express": "6.2.2",
     "@govuk-one-login/frontend-analytics": "1.0.3",
+    "gs": "1.0.0",
```

Comment:
It looks like this package is to do with GhostScript - https://www.npmjs.com/package/gs - so I think it was an accidental install?